edb/schema/delta.py

```diff
@@ -608,6 +608,21 @@ def _modaliases_from_ast(
 
         return modaliases
 
+    @classmethod
+    def _ignorenames_from_ast(
```

Comment:
This isn't an internal method, you're calling it from code that is outside of the class, so trim the leading underscore.

Reply:
Leading underscore removed.